src/Map/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 -   Add method `Symfony\UX\Map\Renderer\AbstractRenderer::tapOptions()`, to allow Renderer to modify options before rendering a Map.
 -   Add `ux_map.google_maps.default_map_id` configuration to set the Google ``Map ID``
+-   Add compatibility with [Live Components](https://symfony.com/bundles/ux-live-component/current/index.html), for the moment only zoom, center, markers and polygons are supported.
```

Comment:
```suggestion
-   Add `ComponentWithMapTrait` to ease maps integration in  LiveComponent.
```

At least, i'd remove the "for the moment..." as it does not bring real value and give some negatiev tonality to the entry :)